src/modules/docker_context.rs

```diff
@@ -0,0 +1,48 @@
+use dirs::home_dir;
+
+use super::{Context, Module, RootModuleConfig};
+
+use crate::configs::docker_context::DockerContextConfig;
+use crate::utils;
+
+const DOCKER_CONFIG_FILE: &str = ".docker/config.json";
+
+/// Creates a module with the currently active Docker context
+///
+/// Will display the Docker context if the following criteria are met:
+///     - There is a file named `$HOME/.docker/config.json`
+///     - The file is JSON and contains a field named `currentContext`
+///     - The value of `currentContext` is not `default`
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module(DOCKER_CONFIG_FILE);
```

Comment:
I think this should be
```suggestion
    let mut module = context.new_module("docker_context");
```